### PR TITLE
fix: getAllCities function keys param converted to optional

### PR DESCRIPTION
**Pull request by NishargShah**
Fixed: https://github.com/harpreetkhalsagtbit/country-state-city/issues/159

### CODE REVIEW

File: src/city.ts [hunk 1]
```diff
@@ -12,13 +12,13 @@ const KEYS = [
 
 let convertedCityList: ICity[] = [];
 // Get a list of all cities.
-function getAllCities(keys: string[]): ICity[] {
+function getAllCities(keys?: string[] = KEYS): ICity[] {
```
Comment (harpreetkhalsagtbit) [inline src/city.ts:15]: getting this err while running tests - `typescript Parameter cannot have question mark and initializer.`

Comment (harpreetkhalsagtbit) [inline src/city.ts:15]:
```suggestion
function getAllCities(keys: string[] = KEYS): ICity[] {
```

**Review by harpreetkhalsagtbit — CHANGES_REQUESTED** [review]
Thanks for raising the fix.
Please check this!

File: src/city.ts (hunk 1 above)
Comment (NishargShah) [reply src/city.ts:15]: Ohh my bad

**Review by harpreetkhalsagtbit — APPROVED** [review]

**Pull request merged**